docs/warmed-containers.md

```diff
@@ -42,7 +42,7 @@ So those can be configured in the same way that operators configure the `invocat
 ```json
 {
   "_id": "guest/limits",
-  "invocationPerMinutes": 10,
+  "invocationsPerMinute": 10,
```

Comment:
This is actually not the main concern of this document but anyway it's a typo.

Reply:
```
So those can be configured in the same way that operators configure the `invocationPerMinute` limit.
```
Should change `invocationPerMinute` to `invocationsPerMinute`